hbase-it/src/test/java/org/apache/hadoop/hbase/test/IntegrationTestBigLinkedList.java

```diff
@@ -122,73 +122,100 @@
  * which is in turn inspired by the Accumulo test called continous ingest (ci).
  * The original source code can be found here:
- * https://github.com/keith-turner/goraci
- * https://github.com/enis/goraci/
- *
+ * <ul>
+ *   <li>https://github.com/keith-turner/goraci</li>
+ *   <li>https://github.com/enis/goraci/</li>
+ * </ul>
+ * </p>
+ * <p>
  * Apache Accumulo [0] has a simple test suite that verifies that data is not
  * lost at scale. This test suite is called continuous ingest. This test runs
  * many ingest clients that continually create linked lists containing 25
  * million nodes. At some point the clients are stopped and a map reduce job is
- * run to ensure no linked list has a hole. A hole indicates data was lost.··
- *
+ * run to ensure no linked list has a hole. A hole indicates data was lost.
+ * </p>
+ * <p>
  * The nodes in the linked list are random. This causes each linked list to
  * spread across the table. Therefore if one part of a table loses data, then it
  * will be detected by references in another part of the table.
- *
- * THE ANATOMY OF THE TEST
+ * </p>
+ * <p>
+ * <h3>THE ANATOMY OF THE TEST</h3>
  *
  * Below is rough sketch of how data is written. For specific details look at
  * the Generator code.
- *
- * 1 Write out 1 million nodes· 2 Flush the client· 3 Write out 1 million that
- * reference previous million· 4 If this is the 25th set of 1 million nodes,
- * then update 1st set of million to point to last· 5 goto 1
- *
+ * </p>
+ * <p>
+ * <ol>
+ * <li>Write out 1 million nodes</li>
+ * <li>Flush the client</li>
+ * <li>Write out 1 million that reference previous million</li>
+ * <li>If this is the 25th set of 1 million nodes, then update 1st set of
+ * million to point to last</li>
+ * <li>goto 1</li>
+ * </ol>
+ * </p>
+ * <p>
  * The key is that nodes only reference flushed nodes. Therefore a node should
  * never reference a missing node, even if the ingest client is killed at any
  * point in time.
- *
+ * </p>
+ * <p>
  * When running this test suite w/ Accumulo there is a script running in
  * parallel called the Aggitator that randomly and continuously kills server
- * processes.·· The outcome was that many data loss bugs were found in Accumulo
- * by doing this.· This test suite can also help find bugs that impact uptime
- * and stability when· run for days or weeks.··
- *
- * This test suite consists the following· - a few Java programs· - a little
- * helper script to run the java programs - a maven script to build it.··
- *
+ * processes. The outcome was that many data loss bugs were found in Accumulo
+ * by doing this. This test suite can also help find bugs that impact uptime
+ * and stability when run for days or weeks.
+ * </p>
+ * <p>
+ * This test suite consists the following
+ * <ul>
+ * <li>a few Java programs</li>
+ * <li>a little helper script to run the java programs</li>
+ * <li>a maven script to build it</li>
+ * </p>
```

Comment:
A closing `ul` seems to be missing, leading to a wrong indentation afterwards in the output.

Reply:
Yes you're right. Looks like checkstyle actually catches these issues, nice!